Documentation/docs/contributing/index.md

```diff
@@ -320,13 +320,15 @@ their branch into logical commits (preferred), maintainers should use the
 *Squash and merge* button on the pull request to consolidate patches into
 logical commits.
 
-For bug fixes that are ready to be included in the next patch release, make a
-comment on the pull request which states the topic should be merged to the
-`release` and `release-X.X` maintenance branch, where `X.X` are the current
-maintenance version, e.g. `5.4`.
+### Merge a bug fix for a patch release
 
-Here are the recommended steps to merge a topic to both `release` and `master`
-branches, assuming the topic branch is forked off the `release` branch:
+
```

Comment:
I think this should clearly state that a topic to be merged for a patched release should be based on `release-X.X` and a PR made into `release-X.X`. Although I'm not 100% what is what should be done.

Reply:
The branch base is documented here:

  https://docs.itk.org/en/latest/contributing/index.html#create-a-topic

It could be changed to `release-X.X`.